rocAL/include/loaders/video/video_loader.h

```diff
@@ -74,10 +74,8 @@ class VideoLoader : public LoaderModule {
     size_t _sequence_length;
     std::thread _load_thread;
     RocalMemType _mem_type;
-    decoded_image_info _decoded_img_info;
-    decoded_image_info _output_decoded_img_info;
     CircularBuffer _circ_buff;
-    TimingDBG _swap_handle_time;
+    TimingDbg _swap_handle_time;
     bool _is_initialized;
     bool _stopped = false;
     bool _loop;                         //<! If true the reader will wrap around at the end of the media (files/images/...) and wouldn't stop
```

Comment:
@swetha097 What files/images are you referring to?

Reply:
Files/ images here refers to video frames